DigitalLearningSolutions.Web/Styles/nhsuk.scss

```diff
@@ -7,3 +7,7 @@
 .nhsuk-hint {
   white-space: initial;
 }
+
+.nhsuk-width-container {

```

Comment:
We should override this in Layout.scss (since nhsuk.css is imported from the NHS Design System and isn't our own stylesheet)

Reply:
fixed